web/src/components/merchant/Routes.tsx

```diff
@@ -0,0 +1,43 @@
+/**
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import React, {lazy} from 'react';
+
+import MerchantProvider from 'components/merchant/contexts/MerchantContext';
+import {useRouteMatch, Switch, Route} from 'react-router-dom';
+
+const MerchantLandingPage = lazy(() => import('components/merchant/landing'));
+const MerchantSignUpPage = lazy(() => import('components/merchant/sign-up'));
+const MerchantSignInPage = lazy(() => import('components/merchant/sign-in'));
+const MerchantOngoingListingsPage = lazy(() =>
+  import('components/merchant/ongoing-listings')
+);
+
+const MerchantRoutes: React.FC = () => {
+  const {path} = useRouteMatch();
+  return (
+    <Switch>
+      <MerchantProvider>
+        <Route exact path={path} component={MerchantLandingPage} />
```

Comment:
@ifhuang So far it's nothing yet. We were thinking this could be a landing page to attract merchants to sign up, something like this https://cloud.google.com/

Reply:
Try not to create things or features that you are not building (or planing to build in the next 3 days) yet. We may end up with a delivery at the end of internship with quite some empty promises/Todos.

Reply:
@xlai20 @karenfrilya97 I think it is not nice to leave `/` as a `404` page either though! If you guys don't like the idea of an empty `MerchantLandingPage` I think for now the easiest thing we can do is smt like:
```suggestion
        <Route exact path={path}>
          <Redirect to=`${path}sign-in` />
        </Route>
```

Reply:
@xlai20 @karenfrilya97 I think it is not nice to leave `/merchant` as a `404` page either though! If you guys don't like the idea of an empty `MerchantLandingPage` I think for now the easiest thing we can do is smt like:
```suggestion
        <Route exact path={path}>
          <Redirect to=`${path}sign-in` />
        </Route>
```

Reply:
@xlai20 Noted. This page was added as a part of the FE structure at the start and I just left it there.
@hzxcaryn I like that idea! Will add that in.